service/api/interfaces.go

```diff
@@ -31,6 +32,7 @@ type UnifiedClient interface {
 	Close()
 	errorHandler
 	StartInterpreterWorkflow(ctx context.Context, options StartWorkflowOptions, args ...interface{}) (runId string, err error)
+	StartWaitForStateCompletionWorkflow(ctx context.Context, options StartWorkflowOptions, args ...interface{}) (runId string, err error)
```

Comment:
We don't need this method here. The workflow should be started by `ApiV1WorkflowWaitForStateCompletion` instead